src/org/exist/http/servlets/XSLTServlet.java

```diff
@@ -124,12 +119,11 @@ private boolean isCaching() {
         return caching;
     }
 
-
     @Override
     protected void doPost(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
 
-        final String stylesheet = (String) request.getAttribute(REQ_ATTRIBUTE_STYLESHEET);
-        if (stylesheet == null) {
+        final String uri = (String) request.getAttribute(REQ_ATTRIBUTE_STYLESHEET);
```

Comment:
probably this should  not be part of the PR ; please keep functional changes and refactoring a bit separated.